app/src/main/java/org/systers/mentorship/viewmodels/MembersViewModel.kt

```diff
@@ -57,6 +58,9 @@ class MembersViewModel : ViewModel() {
                             is IOException -> {
                                 message = MentorshipApplication.getContext()
                                         .getString(R.string.error_please_check_internet)
+                                NetworkStateReceiver.isOnline.observeForever {
+                                    if (it) getUsers(true)
```

Comment:
I don't think the parameter passed should be true always, only if the user has refreshed it , the parameter should be true.

Reply:
done